src/Editor/Resources.cpp

```diff
@@ -22,49 +22,55 @@ ResourceList& ResourceList::GetInstance() {
 }
 
 void ResourceList::Save() const {
+    // Save to file.
+    ofstream file(GetSavePath());
+    file << ToJson();
+    file.close();
+}
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'activeScene' is assigned in constructor body. Consider performing initialization in initialization list. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuseInitializationList)